src/main/java/com/revealprecision/revealserver/persistence/domain/EntityTags.java

```diff
@@ -0,0 +1,45 @@
+package com.revealprecision.revealserver.persistence.domain;
+
+import java.util.UUID;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.FieldNameConstants;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+import org.hibernate.envers.Audited;
+
+@FieldNameConstants
+@Entity
+@Audited
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@SQLDelete(sql = "UPDATE condition SET entity_status = 'DELETED' where identifier=?")
+@Where(clause = "entity_status='ACTIVE'")
+public class EntityTags extends AbstractAuditableEntity {
```

Comment:
is this a collection of Entities , if it's called EntityTags, instead of EntityTag? We can rename the enitity and call it EntityTag and still leave the database table as entity_tags. We will need to talk about this. I also saw the same on PlanAssignment vs PlanAssignments .

Reply:
Agree....will resolve this.....i think for completeness i will update the DB as well....will there be a concern about that?

Reply:
all good